cfme/tests/infrastructure/test_vm_rest.py

```diff
@@ -177,3 +178,36 @@ def test_filtering_vm_with_multiple_ips(appliance, provider):
     )
     assert_response(appliance)
     assert vm.name in [resource.name for resource in result.resources]
+
+
+@pytest.mark.meta(automates=[1581853])
+@pytest.mark.tier(3)
+@pytest.mark.provider(classes=[InfraProvider], selector=ONE)
+def test_database_wildcard_should_work_and_be_included_in_the_query(appliance, request, provider):
+    """ Database wildcard should work and be included in the query
+    Bugzilla:
+        1581853
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Rest
+        testtype: functional
+        initialEstimate: 1/4h
+        startsin: 5.10
+        testSteps:
+            1. Create a VM with some name, for e.g test-25-xyz.
```

Comment:
For this test and others in the module, its unnecessary to create a VM if there is one that exists on the provider already that has been inventoried by MIQ.

Not required for this PR, but consider using a module scoped fixture to provide the VM for this test and others within this module that don't actually make any modifications to the VM. The fixture can save a lot of wasted time in these test functions by just using an existing VM.